index.ts

```diff
@@ -43,3 +46,25 @@ server.on('listening', (): void => {
     logger.info('http server listening on', conf.get('port') );
 });
 server.on('after', util.after);
+
+// Socket.IO
+if (conf.get('websocket.socket-io.enable')) {
+    var serverSio: any = restify.createServer(conf.get('serverOptions'));
+    serverSio.listen(conf.get('webSocket.socket-io.port'));
+    serverSio.on('listening', (): void => {
+        logger.info('socket.io server listening on', conf.get('webSocket.socket-io.port') );
+    });
+    var io: SocketIO.Namespace = sio(serverSio.server).of('/subscription');
+    io.on('connection', webSocketHandler.sioOnConnect);
+}
+
+// ws
+if (conf.get('websocket.ws.enable')) {
+    var serverWS: any = restify.createServer(conf.get('serverOptions'));
+    serverWS.listen(conf.get('webSocket.ws.port'));
```

Comment:
We can have this discussion later, but it may be convenient to have the the `ws`, standard WebSocket protocol, share the same port as the http server (or have it be configurable that way).